Adding images to data sources page

JanProgrammierung opened the pull request:
- Created public folder and added the images from NextCloud (renamed)
- Changed the src and img element as well as the parent div to have a better design
- Changed alphabetical order of disease types filter list

SevLG commented on pmp-frontend-app/src/pages/AboutProductPage.tsx:
Could this be moved to our content.ts file? Would be more consistent and easy to find later if keep all stuff like this there.

JanProgrammierung replied:
I agree. See comment in other PR.

SevLG replied:
I answered in the other PR, but for consistency I'll write here as well. We can probably look at a more consistent way to do it later cause there would be issues with links and such. We'll keep it as it is for now.

SevLG commented on pmp-frontend-app/src/components/DataSourcesComponent.tsx:
```diff
@@ -51,8 +51,8 @@ export default function DataSourcesComponent(): ReactElement {
 
-    const dataSourcesURI: string = 'https://raw.githubusercontent.com/ScilifelabDataCentre/data.scilifelab.se/main/data/data_sources.json';
-    // const dataSourcesURI: string = 'https://raw.githubusercontent.com/SevLG/data.scilifelab.se/patch-1/data/data_sources.json';
+    // const dataSourcesURI: string = 'https://raw.githubusercontent.com/ScilifelabDataCentre/data.scilifelab.se/main/data/data_sources.json';
```
Liane told us to use their develop branch instead of their main branch, so we don't have to wait for them to do releases to get the updated JSON.

Try using this one instead:
https://raw.githubusercontent.com/ScilifelabDataCentre/data.scilifelab.se/develop/data/data_sources.json

SevLG reviewed (changes requested):
Looks good code wise but had a few comments (hopefully you can see them this time).

Also a few of the thumbnails look a bit off to me, but I don't know if egregious enough to worry about fixing. I don't think it's a huge deal, what do you think? I'll attach screenshots below.

Big difference in size of almost the same logo looks a bit wonky:
<img width="835" alt="Screenshot 2024-05-25 at 18 09 52" src="https://github.com/ScilifelabDataCentre/precision-medicine-portal-frontend/assets/49151393/91a87510-b154-40b5-a875-92a7cb33a49a">

Unusually large gap to the right of the thumbnail compared to the other thumbnails:
<img width="893" alt="Screenshot 2024-05-25 at 18 10 18" src="https://github.com/ScilifelabDataCentre/precision-medicine-portal-frontend/assets/49151393/4b363e3d-5f14-4173-b7b3-38016a151b13">

Looks a bit weird how the long text lines up with the logo:
<img width="828" alt="Screenshot 2024-05-25 at 18 10 03" src="https://github.com/ScilifelabDataCentre/precision-medicine-portal-frontend/assets/49151393/c3bba6fb-9d0f-4842-9a95-7de2d973656e">

JanProgrammierung commented:
> Looks good code wise but had a few comments (hopefully you can see them this time).
> 
> Also a few of the thumbnails look a bit off to me, but I don't know if egregious enough to worry about fixing. I don't think it's a huge deal, what do you think? I'll attach screenshots below.
> 
> Big difference in size of almost the same logo looks a bit wonky: <img alt="Screenshot 2024-05-25 at 18 09 52" width="835" src="https://private-user-images.githubusercontent.com/49151393/333817441-91a87510-b154-40b5-a875-92a7cb33a49a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MjE0ODIsIm5iZiI6MTcxNjgyMTE4MiwicGF0aCI6Ii80OTE1MTM5My8zMzM4MTc0NDEtOTFhODc1MTAtYjE1NC00MGI1LWE4NzUtOTJhN2NiMzNhNDlhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDE0NDYyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxOTI4YzBkMTM3MTFjZjVmYTM4ZWUzMjhkZDYxNzdiMWEyZTU0NmMyZGM4YzczNDhkOWVjNTZiMjMzNTZiNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ddwwCD-BPUhyAcl9M-T-1QFJh1T7aUFVhXgYb8TqG_A">
> 
> Unusually large gap to the right of the thumbnail compared to the other thumbnails: <img alt="Screenshot 2024-05-25 at 18 10 18" width="893" src="https://private-user-images.githubusercontent.com/49151393/333817447-4b363e3d-5f14-4173-b7b3-38016a151b13.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MjE0ODIsIm5iZiI6MTcxNjgyMTE4MiwicGF0aCI6Ii80OTE1MTM5My8zMzM4MTc0NDctNGIzNjNlM2QtNWYxNC00MTczLWI3YjMtMzgwMTZhMTUxYjEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDE0NDYyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxMjI4ZWZmZjNkYmY1NGU5Y2U0ODZiMjc0ODEwMTBjMjI1ZTZjZDIyYmM5NDhmZmM1MTQ5NzcwMGFlYTZkZDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5je6XNi4ssHRHt_llvEewQTSJY9T8v8zzgRByV7cB4s">
> 
> Looks a bit weird how the long text lines up with the logo: <img alt="Screenshot 2024-05-25 at 18 10 03" width="828" src="https://private-user-images.githubusercontent.com/49151393/333817444-c3bba6fb-9d0f-4842-9a95-7de2d973656e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4MjE0ODIsIm5iZiI6MTcxNjgyMTE4MiwicGF0aCI6Ii80OTE1MTM5My8zMzM4MTc0NDQtYzNiYmE2ZmItOWQwZi00ODQyLTlhOTUtN2RlMmQ5NzM2NTZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDE0NDYyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxMjM3YzIwMzc4YmM0OGRmM2M5OTUxOWU5OGU5ZWI3NDY3MjY1ZGUwNGI4YWViMWQ3ZWMwOTI0NmNmNmU4OWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.K528XsT5wuicT7xG_IkU_8xT8kwYAi1tav3snn3Mvlw">

Yes, they are not optimal. I already edited lots of the images to take whitespace on the right side away and have them align better.
The "Data Commons" images: The official logos are in a different size. I tried to resize one of them but then it stretches and looks very weird. It might be fixed with some extra work but maybe can be done in the future when there's more time?
The large gap: Fixed it by adding object-right.
The img wrap: Fixed it by adding margin left 2.
